sdk/face/azure-ai-vision-face/README.md

```diff
@@ -0,0 +1,429 @@
+List<String> faceIds = detectResults.stream().map(FaceDetectionResult::getFaceId).collect(Collectors.toList());
+```
+
+When the training operation is completed successfully, we can identify the faces in this LargePersonGroup through
+`FaceClient`.
+```java com.azure.ai.vision.face.readme.faceRecognitionFromLargePersonGroup.identify
+// Stage 3: identification
+List<FaceIdentificationResult> results = client.identifyFromLargePersonGroup(faceIds, largePersonGroupId);
+for (FaceIdentificationResult result : results) {
+    System.out.println("Identification result for " + result.getFaceId());
+    for (FaceIdentificationCandidate identificationCandidate : result.getCandidates()) {
+        System.out.println("Found:" + identificationCandidate.getPersonId() + ", confidence: " + identificationCandidate.getConfidence());
+    }
+}
+```
+
+Finally, use `FaceAdministrationClient` to remove the large person group if you don't need it anymore.
+```java com.azure.ai.vision.face.readme.faceRecognitionFromLargePersonGroup.deleteLargePersonGroup
+administrationClient.deleteLargePersonGroup(largePersonGroupId);
+```
+
+### Liveness detection
+Face Liveness detection can be used to determine if a face in an input video stream is real (live) or fake (spoof).
+The goal of liveness detection is to ensure that the system is interacting with a physically present live person at
+the time of authentication. The whole process of authentication is called a session.
+
+There're two different components in the authentication: a mobile application and an app server/orchestrator.
+Before uploading the video stream, the app server has to create a session, and then the mobile client could upload
+the payload with a `session authorization token` to call the liveness detection. The app server can query for the
+liveness detection result and audit logs anytime until the session is deleted.
+
+The Liveness detection operation can not only confirm if the input is live or spoof, but also verify whether the input
+belongs to the expected person's face, which is called **liveness detection with face verification**. For the detail
+information, please refer to the [tutorial][liveness_tutorial].
+
+We'll only demonstrates how to create, query, delete a session and get the audit logs here. For how to perform a
+liveness detection, please see the sample of [mobile applications][integrate_liveness_into_mobile_application].
+
+Here is an example to create and get the liveness detection result of a session.
+```java com.azure.ai.vision.face.readme.createLivenessSessionAndGetResult
+System.out.println("Create a liveness session.");
+FaceSessionClient sessionClient = new FaceSessionClientBuilder()
+    .endpoint(endpoint)
+    .credential(new KeyCredential(accountKey))
+    .buildClient();
+
+CreateLivenessSessionContent parameters = new CreateLivenessSessionContent(LivenessOperationMode.PASSIVE)
+    .setDeviceCorrelationId(UUID.randomUUID().toString())
```

Comment:
question: what's a DeviceCorrelationId for?

Reply:
Device correlation id is provided by customer's client side app. It helps customer to identify the client device sending the liveness request. Please refer to https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness#orchestrate-the-liveness-solution for complete scenrio.

Reply:
There are two components for our customers to use our liveness check API . They need to create a server application , and  client mobile application.
The DeviceCorrelationId  is the ID of a clinet mobile device. This ID can be generated by app server  or provided by a mobile device depending Customers' architectures.

Reply:
Then, I kind of assume the same `deviceCorrelationId` value is to be reused by client in future request.

If so, in sample, should we do an explicit var like
```
String deviceCorrelationId = UUID.randomUUID().toString();
```
as a guide that this to be persisted (at least in memory) and re-used?

Reply:
Good suggestion. Updated.